client/src/modules/inventory/inventory.routes.js

```diff
@@ -30,6 +30,9 @@ angular.module('bhima.routes')
       })
       .state('inventory.create', {
         url : '/create',
+        params : {
+          creating : { value : true }
+        },        
```

Comment:
It makes it possible to differentiate the creation of the update, to consider only the present of the $state.params.uuid was not sufficient especially when one refreshed the browser